bundles/org.openhab.binding.allplay/src/main/resources/OH-INF/i18n/allplay.properties

```diff
@@ -1,18 +1,18 @@
-# binding
+# add-on
 
-binding.allplay.name = AllPlay Binding
-binding.allplay.description = The AllPlay binding integrates devices compatible with Qualcomm AllPlay.
+addon.allplay.name = AllPlay Binding
+addon.allplay.description = The AllPlay binding integrates devices compatible with Qualcomm AllPlay.
 
-# binding config
+# add-on config
 
-binding.config.allplay.callbackUrl.label = Callback URL
-binding.config.allplay.callbackUrl.description = URL to use for playing audio streams, e.g. http://192.168.0.2:8080
-binding.config.allplay.fastForwardSkipTimeInSec.label = Fast Forward Skip Time
-binding.config.allplay.fastForwardSkipTimeInSec.description = Seconds to jump forward if the fastforward command is executed
-binding.config.allplay.rewindSkipTimeInSec.label = Rewind Skip Time
-binding.config.allplay.rewindSkipTimeInSec.description = Seconds to jump backwards if the rewind command is executed
-binding.config.allplay.zoneMemberSeparator.label = Zone Member Separator
-binding.config.allplay.zoneMemberSeparator.description = Separator which is used when sending multiple zone members to channel 'zonemembers'
+addon.config.allplay.callbackUrl.label = Callback URL
```

Comment:
Is this correct?

Reply:
IMO yes, but I‘m no expert. It‘s configuration at add-on level.

Reply:
I'll compare with i18n tool output later.

Reply:
You are right, same is generated by `i18n:generate-default-translations`.